### PR TITLE
deflake sync suite

### PR DESCRIPTION
**Pull request by ben-c-at-moz**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Bugzilla bug ID

**Link:** https://bugzilla.mozilla.org/show_bug.cgi?id=1906920

## Type of change

- [x] Other Changes - Deflake sync suite

# How does this resolve / make progress on that bug?

Test passes in CI

# Comments / Concerns

Not certain why it works now

### CODE REVIEW

**Review by sarinali — APPROVED** [review]
lgtm, stuff resolved from last time, both pass

**Review by sarinali — APPROVED** [review]
lgtm merge conflicts suck

**Review by sarinali — DISMISSED** [review]
good callout for screenshot!

**Comment by ben-c-at-moz** [comment]
For quantum reasons :), this test only works in CI if I take a picture of it.

File: tests/sync_and_fxa/test_existing_fxa.py [hunk 1]
```diff
@@ -20,10 +20,14 @@ def fxa_test_account():
...
 
 
-@pytest.mark.unstable
 def test_sync_existing_fxa(
-    driver: Firefox, fxa_test_account: Tuple[str, str], restmail_session, get_otp_code
+    driver: Firefox,
+    fxa_test_account: Tuple[str, str],
+    restmail_session,
+    get_otp_code,
+    screenshot,
```
Comment (sarinali) [inline tests/sync_and_fxa/test_existing_fxa.py:28]: 💀 it be liek that

**Review by sarinali — APPROVED** [review]
LGTM

**Pull request merged**